img/folder.svg

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I don't see where those svg are used? :)

Reply:
yes it is used, see here
![image](https://user-images.githubusercontent.com/9105243/74089215-41a7e880-4a9f-11ea-9046-adb93323f39d.png)

Reply:
Please use our existing icons then :)
https://docs.nextcloud.com/server/stable/developer_manual/design/icons.html

Reply:
The other icons on that page are also in `apps/serverinfo/img`. Is it possible to use the core icons with imagepath? Like `image_path('core', 'icon-files.svg')`?

Reply:
should be, but please move to class icons so it is dark-theme compatible

Reply:
Removed all .svg icons from the img folder except the app-icon and embeded the core icons.

Just have to say – it is not very clear which "path" you have to set for each icon shown in https://docs.nextcloud.com/server/stable/developer_manual/design/icons.html

Sometimes it is "actions", sometimes "categories" and sometimes "places". I had to look for the icon in the core images folder to know which paths I had to use. Not very intuitive.

The result now looks something like that:
![image](https://user-images.githubusercontent.com/9105243/75194708-f22d1200-5758-11ea-87d1-8e86a32fbfaa.png)
![image](https://user-images.githubusercontent.com/9105243/75194746-ffe29780-5758-11ea-8d22-14c2463f31a8.png)
![image](https://user-images.githubusercontent.com/9105243/75194768-0c66f000-5759-11ea-9101-3c1c785e4e2b.png)

Reply:
I guess we can still migrate to icon class later.

Reply:
> Just have to say – it is not very clear which "path" you have to set for each icon shown in [docs.nextcloud.com/server/stable/developer_manual/design/icons.html](https://docs.nextcloud.com/server/stable/developer_manual/design/icons.html)

This is because that is not the point of this documentation. You should not use img directly, but use icon classes ;)

Reply:
File: `style.scss`
```
h2 > span[class^='icon-'], h2 > span[class*=' icon-'] {
	padding-left: 30px;
}

@include icon-black-white('quota', 'actions', 1, true);
```
File: `settings-admin.php`
```
<h2>
        <span class="icon-quota"></span>
        <?php p($l->t('Disk')); ?>
</h2>
```

That seems to work. There are probably better ways ;) Just to give you some points how to start. Please keep in mind to enable `'debug'  => true` in `config.php` to disable caching. If you have the impression that some resources are cached try `php occ maintenance:repair`